src/patterns/news/index.njk

```diff
@@ -0,0 +1,114 @@
+---
+title: News
+group: Pages
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/lists/_macro.njk" import onsList %}
+
+How to display, tag, categorise and provide a landing page preview for news articles.
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/landing/index.njk" })
+}}
+
+## When to use this pattern
+
+Use this pattern to upload a news article to a public-facing website.
+
+## How to use this pattern
+
+Each article needs to have a:
+
+- heading
+- summary
+- body text
+- category
+- date
+
+Use the [articles component](/components/articles) to build consistent articles.
+
+The content needs to be clearly written and as short as possible. Use plain language and “frontload” – put the most important word or points first. This ensures that if the user is skimming the article, they read the most important thing and understand it. 
+
+Our house style guide has [advice on writing in plain English](https://style.ons.gov.uk/category/writing-for-the-web/plain-english). You could also ask a content designer to help you rewrite the content so it is clear and helpful for the user.
+
+### Landing page
+
+This page sorts the already-published news so that the most recently posted article is at the top of the list.
+
+Each landing page preview needs to have a:
+
+- short, clearly written title that links to the news article
+- short summary, usually taken from the first paragraph of the article, which should be no more than four lines (around 150 characters)
+- [date that follows house style](https://style.ons.gov.uk/category/writing-for-the-web/plain-english), for example, 23 September 2020
+- category, which links to the page that lists all articles in that category
+- thumbnail image
+
+You can add [related content](https://ons-design-system.netlify.app/components/related-content), which appears on the right-hand side of the landing page. Related content could include a list of categories and/or contact details for the media team. Another optional feature is “[Follow us](https://ons-design-system.netlify.app/styles/typography/#list-with-social-icons)” social media links to Instagram, Facebook, Twitter and LinkedIn.
+
+Add an image to your landing page. If you do not include an image, the system will use a placeholder. Placeholders do not look professional or support the written content to tell a story. The [articles component](/components/articles) has advice on image dimensions.
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/landing/index.njk" })
+}}
+
+### Article
```

Comment:
I agree it can get confusing between the article pattern and component. But do we use "snippet" anywhere else. What makes a component a snippet? Might need to consider this alongside other components

Reply:
"Snippet" is just a term used in SEO which refers to the title/link and description of a page shown in a "results" view like this. 
https://yoast.com/what-is-a-snippet/

Reply:
@LauraC119 is looking into this.